00-project-documentation/communication/reports/turing-health-quartery-updates/Turing-final-report.md

```diff
@@ -0,0 +1,76 @@
+
+The project had four goals: (1) Collect data to improve our understanding of sensory processing in people’s daily lives (2) Allow people to share their experiences and coping strategies to support others (3) Give non-autistic people a way to learn about the experiences of autistic people, allowing them to better understand the challenges and provide better support (4) Allow this data to be used by policy makers and creators of spaces and environments, to improve inclusion for autistic people.
+
+## Lessons learned
+
+One main lesson from the AutSPACEs project is the potential that deep, engaged collaboration between academic researchers, open source developers and a community of volunteer contributors has for shaping and implementing a research project, in particular in a research environment that often is shaped by mutual distrust between "researchers" and "human subject". By approaching each other on eye level, AutSPACEs managed to build a deep and sustained trust, leading to better and more ethical research than usually observed - especially in the fields of data science and AI. 
+
+Beyond the positive aspect, the project also had to recognize that this approach comes with its own challenges. It requires a high level of engagement and buy-in from researchers - which not all scientists are willing to do. Due to the deliberative nature, this approach also moves slower than the typical academic, top-down governed research project where no deliberation takes place.
+
+The flip side of this is that the deep deliberation can pre-empt challenges, which otherwise could become serious road blocks down the line: For example, the ethics and data protection reviews done within the Turing benefited dramatically from our deliberate approach, as we collectively not only considered a wide range of potential risks and harms to participants and how to protect their data, but also could demonstrate how we deliberatively agreed on tradeoffs in these situations.
+
+## Achievements
+
+There a number of achievements across domains, given the interdisciplinary nature of the project. 
+
+### Citizen Science / Community engagement
+
+- Over its lifetime, AutSPACEs managed to engage 100s of contributors, including autistic contributors, contributors who are parents/caretakers of autistic people, autism researchers, web/graphic designers, professional writers, open source developers, and many more - including virtually all intersections of these. Our diverse community, and the connections made within it, are one of the most lasting achievements that goes far beyond any traditional research. 
+- Developing a wide range of modes of engagement. Contributors to AutSPACEs were engaged in a wide range of modes, including 
+    - in-person focus groups and 
+    - monthly community meetings, 
+    - user testing and actively interpreting the findings of those tests
+    - co-leading on data interpretations and content moderation design
```

Comment:
```suggestion
    - Co-leading on data interpretations and content moderation design
```
Could we add links to any outputs from these?

Reply:
see outputs at end